recipe/meta.yaml

```diff
@@ -66,9 +66,10 @@ outputs:
         - libcurand-dev      # [(cuda_compiler_version or "").startswith("12")]
       run:
         - python
-        - numpy
+        - numpy >=1.17.0
         - {{ pin_compatible('scipy') }}
         - tqdm >=4.27
+        - threadpoolctl
```

Comment:
nice! thanks for adding this in (I had to add this dependency to fix a bug that we were hitting with numpy 1.26).